### ISSUE TITLE
valorant 64bit version still missing

### ISSUE THREAD

**Issue by Cr4shOv3rrid3** [M-Reimer/wine-lol#70]
i made a pull request for this already.
can you also build a 64bit version as well, as this would solve the valorant whole linux gaming not working at all problem i bet.
thx for lol version by the way ;)
-cr4sh

**Comment by M-Reimer** [comment]
At first: The goal of this (specially patched) wine version is only to make League Of Legends work on Linux.

Second: Valorant does not and will not (for the foreseeable future) work at all on Linux as they use the "famous" kernel level anti cheat which does not run (currently) on Linux.

https://eune.leagueoflegends.com/en-pl/news/dev/dev-null-anti-cheat-kernel-driver/

**Comment by Cr4shOv3rrid3** [comment]
i did not ask you to give me an anti cheat link from riot games

i asked you to build a 64 bit wine patch to get valorant 2 run on riots game servers.

i am interested into a way to get to play this thing on linux and should they running anti cheat problems it would be no big deal to integrate this into the linux patch as well.
what about getting into a copy of this anticheat thing. cracking it open. making it compatible with linux and patch it into this new 64bit wine version?
put a suited guy inside riots company that deals for the management for this and there won't be no more problem with cheaters than they already running with windows. as the OS is clearly not the problem here.

what enables the linux os users to play this game in a normal way. i mean i am interested into playing by the rules not to cheat inside playing the game. this is stupid anyways. destroys the whole fun of gaming online with others. especially for online gaming. 

so i only will ask you one last time:

are you interested into helping me out with this or not? cause you running the necessary expertise towards wine.
you could become the perfect guy for the job as well for help riot with the anti cheat problem for the linux side.
just contact em. on your side.
i will try to do the same in my.

the kernel for windows is an endless loop anyways basically an illusion. this thing was based on QDOS
gates bought it for 50k USD from the guy that build QDOS originally to start all this. and let his main programmer Paul Allan (RIP) change it a bit.
and created Microsoft as his company out of it.
till today this is kernel ring 0 this is connected to. what still leads to gates himself. his physical body.
well to be fair more towards the dead coder than him. as gates was more the business oriented guy in this.
cause without QDOS MSDOS and later microsoft could never been build up in the first place.
this also leads Technically ring 0 into QDOS first developer: https://en.wikipedia.org/wiki/Tim_Paterson

now you got the information.

besides the fact it at the moment is around the course price for 1 Bitcoin what QDOS originally did microsoft cost. so much for coincidence

are you in or not?

cause i am not interested into naysayers i am interested into guys wanna help building up a meaningful future with meaningful stuff. and solid linux integrations for gaming everywhere is an important aspect to this.

**Comment by Cr4shOv3rrid3** [comment]
i give you additional info as well:

https://en.wikipedia.org/wiki/CP/M (1974)

the source towards QDOS and another guy: https://en.wikipedia.org/wiki/Gary_Kildall (working for Digital Research inc.)
and towards that source: Initially confined to single-tasking on 8-bit processors and no more than 64 kilobytes of memory, later versions of CP/M added multi-user variations and were migrated to 16-bit processors.

now technically i halved the 32bits your lol patch works for.
i hope we don't need to go deeper back in history towards OS development here and it should solve the issue!

**Comment by M-Reimer** [comment]
And one last time from me: It does not work on Linux because of the kernel level anti cheat. This is by intention from Riot. And Riot won't do much for Linux users there as the market is way too small. There is an ongoing tendency of cheats that can't be stopped even on kernel level which will ultimately force publishers to do their anti cheat stuff on their servers and no longer on user's clients. I guess we'll have to wait for this to happen before we can expect to get this to run on Linux.

The patch to make LoL work is not from me. I just build the wine version with the patch and offer it to the community. To make LoL work not much is needed actually as they don't rely on kernel level anti cheat there. I am no Wine developer and have absolutely no insights into how wine works internally!

Actually what you should do is to just play the original game where Riot copied from. CS:GO runs natively on Linux without any Wine compatibility hacks.
